### PR TITLE
Update Cookbook.md to add comment about loading 3rd party CSS

### PR DESCRIPTION
**Pull request by bugzpodder**
Inspired by https://github.com/SpareBank1/designsystem/blob/master/styleguide.config.js#L93

### CODE REVIEW

File: docs/Cookbook.md [hunk 1]
```diff
@@ -201,6 +201,8 @@ module.exports = {
 }
 ```
 
+You may also want to use MiniCssExtractPlugin.loader in conjunction when loading 3rd-party CSS.
```
Comment (sapegin) [inline docs/Cookbook.md]: This sentence doesn't say anything without an example.

Comment (sapegin) [reply docs/Cookbook.md]: And the main question: why would you want to use it in a style guide?

**Comment by bugzpodder** [comment]
For example, if I want to load css for font-awesome from node_modules.
Putting it in require section as described does not work as it is not automatically loaded on every page.
https://github.com/styleguidist/react-styleguidist/issues/591
and the other example I referenced have their use cases for this as well, ie:
https://github.com/SpareBank1/designsystem/blob/master/styleguide.config.js#L93

**Comment by bugzpodder** [comment]
ie this is basically adding <link rel="stylesheet" href="path/to/local/style.css" /> as adding the plugin will include the css as part of the build.

**Comment by sapegin** [comment]
CSS extraction an CSS loading are completely independent things, you don't need to extract CSS to a file to do anything, you've described.

**Comment by sapegin** [comment]
I advice you to read what MiniCssExtractPlugin actually does and how CSS imports work in webpack instead of complaining here.

**Pull request closed without merging**